services/device-registry-jdbc/src/test/resources/container-license-acceptance.txt

```diff
@@ -0,0 +1 @@
+mcr.microsoft.com/mssql/server:2017-CU12
```

Comment:
I have no idea what this means.

Reply:
https://www.testcontainers.org/modules/databases/mssqlserver/ explains it. I'll remove that file and use the code based method for accepting the Docker container EULA.